packages/fxa-settings/src/pages/InlineRecoveryKeySetup/index.tsx

```diff
@@ -21,6 +22,25 @@ export const InlineRecoveryKeySetup = ({
   createRecoveryKeyHandler: () => Promise<void>;
   currentStep: number;
 } & RouteComponentProps) => {
+  const doLaterHandler = () => {
```

Comment:
Clicking on the dismiss button from the inline flow should set a separate key from the the dismiss button in the settings promo. Dismissing one shouldn't dismiss both - we want to be able to show both and measure engagement/dismissal rates.